src/pages/Florist/index.tsx

```diff
@@ -56,13 +56,13 @@ const useStyles = makeStyles((theme: Theme) =>
       flexDirection: "column",
       alignItems: "center",
       backgroundColor: "#6ac69780",
-      borderRadius: "4px"
+      borderRadius: "4px",
     },
     boughtName: {
       minHeight: "40px",
       textAlign: "center",
       marginTop: "5%",
-      fontWeight: "bold"
+      fontWeight: "bold",
     },
```

Comment:
Maybe add an object here for if user balance is 0 with the same styling as welcomeText but also include "backgroundColor: theme.palette.error.light"